tests/integration/tables/bam.cpp

```diff
@@ -0,0 +1,32 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed in accordance with the terms specified in
+ *  the LICENSE file found in the root directory of this source tree.
+ */
+
+#include <osquery/tests/integration/tables/helper.h>
+
+namespace osquery {
+namespace table_tests {
+class BamTest : public testing::Test {
+ protected:
+  void SetUp() override {
+    setUpEnvironment();
+  }
+};
+
+TEST_F(BamTest, test_sanity) {
+  // QueryData const rows = execute_query("select * from bam");
```

Comment:
You'll want to uncomment this, if you don't expect `bam` to always produce data you can place the `validate_rows` call in a conditional that checks `if (!rows.empty()) {`